cpp/libcudf_kafka/cmake/thirdparty/CUDF_KAFKA_GetCUDF.cmake

```diff
@@ -40,7 +40,16 @@ function(find_and_configure_cudf VERSION)
                         "BUILD_BENCHMARKS OFF")
     cudfkafka_restore_if_enabled(BUILD_TESTS)
     cudfkafka_restore_if_enabled(BUILD_BENCHMARKS)
+    if(cudf_ADDED)
+        set(cudf_ADDED TRUE PARENT_SCOPE)
+    endif()
 endfunction()
 
-set(CUDF_KAFKA_MIN_VERSION_cudf 0.19)
+set(CUDF_KAFKA_MIN_VERSION_cudf 0.20)
```

Comment:
What about this?
```suggestion
set(CUDF_KAFKA_MIN_VERSION_cudf "${CUDF_KAFKA_VERSION_MAJOR}.${CUDF_KAFKA_VERSION_MINOR}")
```

Reply:
Should this also be updated as part of the `ci/release/update-version.sh` file?

Reply:
I think the suggested change is the proper approach as that reduces the amount of places `ci/release/update-version.sh` needs to update.